### PR TITLE
feat(server-config): document new server CLI flag egress-selector-mode

### PR DESCRIPTION
**Pull request by dlouzan**
Related https://github.com/k3s-io/k3s/pull/5603

### CODE REVIEW

File: docs/installation/install-options/server-config.md [hunk 1]
```diff
@@ -20,13 +20,13 @@ In this section, you'll learn how to configure the K3s server.
     - [Listeners](#listeners)
     - [Data](#data)
     - [Networking](#networking)
-    - [Customized Options](#customized-options)
```
Comment (dlouzan) [inline docs/installation/install-options/server-config.md:23]: This anchor is currently broken

File: docs/installation/install-options/server-config.md [hunk 2]
```diff
@@ -147,6 +147,7 @@ the agent options are there because the server has the agent process embedded wi
 |  `--cluster-dns` value   | "10.43.0.10"         | Cluster IP for coredns service. Should be in your service-cidr range |
 |  `--cluster-domain` value  | "cluster.local"        | Cluster Domain       | 
 |  `--flannel-backend` value   | "vxlan"      | One of 'none', 'vxlan', 'ipsec', 'host-gw', or 'wireguard'       |
+|  `----egress-selector-mode` value | "pod" | One of 'agent', 'cluster', 'pod', or 'disabled' |
```
Comment (dlouzan) [inline docs/installation/install-options/server-config.md:150]: As a user, I'd honestly value having the explanations in the description of https://github.com/k3s-io/k3s/pull/5603, but not sure how this fits with your refactoring work on https://github.com/k3s-io/docs/pull/6

**Comment by dereknola** [comment]
Closing this PR as the changes have been merged in #6

**Pull request closed without merging**